src/Utilities/Timer.hh

```diff
@@ -63,12 +63,12 @@ public:
 
   static std::list<Timer*> TimerList;
 
-  static void TimerSummary(const int bert, const int ernie) {
+  static void TimerSummary(const std::string& fname);
+  static void TimerSummary(){ TimerSummary("time.table"); }
+  static void TimerSummary(const int, const int) {
     TimerSummary(); // backwards compatibilty...
   }
   
-  static void TimerSummary(void);
```

Comment:
@jmikeowen I know passing void as an argument type is a C trick for explicitly defining a function takes 0 arguments. Does removing this break some interoperability with the C wrapping library that I may not be aware of?

Reply:
I don't think you're breaking anything there.  This Timer class stuff is a really old version of something constructed by Noel Keen years ago, who now works at Lawrence Berkeley Lab still on performance stuff.  You could ask him (NDKeen@lbl.gov) for more info on why he did it this way, and he likely has some updated version of this by now.